<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutDialog</class>
 <widget class="QDialog" name="ShortcutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Shortcut</string>
  </property>
  <property name="whatsThis">
   <string>Press your shortcut keys and select OK</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="shortcutsGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="primaryButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Select to change the primary keyboard shortcut</string>
        </property>
        <property name="text">
         <string>Primary Shortcut:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="primaryClearButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Press to clear the key sequence buffer.</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="keyEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="alternateButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Select to change the alternative keyboard shortcut</string>
        </property>
        <property name="text">
         <string>Alternative Shortcut:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="alternateClearButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Press to clear the key sequence buffer.</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="alternateKeyEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>keyEdit</tabstop>
  <tabstop>alternateKeyEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>primaryButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>primaryClearButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alternateButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>alternateClearButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShortcutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
